pkg/util/mount/mount_linux.go

```diff
@@ -240,13 +248,31 @@ func readProcMountsFrom(file io.Reader, out *[]MountPoint) (uint32, error) {
 func (mounter *SafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
 	options = append(options, "defaults")
 
+	// Run fsck on the disk to fix repairable issues
+	args := []string{"-a", source}
+	cmd := mounter.Runner.Command("fsck", args...)
```

Comment:
> You might want to do something similar to what nsenter_mount.go does to find the absolute path to the fsck binary

Hmm.. I don't think that is necessary. `fsck` is a low level utility and is usually found in PATH. In addition this PR handles the case where `fsck` is not found and proceeds with a mount after logging a warning. but if something like is is really needed then I think it should be done in `pkg/util/exec` so all callers can benefit.

Incidentally, it looks like nsenter_mount is overriding $PATH which might confuse some users.